### PR TITLE
Add ZooKeeper Ops Request

### PR DESCRIPTION
**Pull request by AbdullahAlShaad**

### CODE REVIEW

**Review by raihankhan — CHANGES_REQUESTED** [review]
Fix comments. comments have references to rabbitmq, druid etc.

File: apis/ops/v1alpha1/zookeeper_ops_helpers.go [hunk 1]
```diff
@@ -27,50 +27,50 @@ import (
 	"kmodules.xyz/client-go/apiextensions"
 )
 
-func (r *ZooKeeperOpsRequest) CustomResourceDefinition() *apiextensions.CustomResourceDefinition {
+func (s *ZooKeeperOpsRequest) CustomResourceDefinition() *apiextensions.CustomResourceDefinition {
```
Comment (raihankhan) [inline apis/ops/v1alpha1/zookeeper_ops_helpers.go:30]: Maybe use `z *ZooKeeperOpsRequest` as receiver.

File: apis/ops/v1alpha1/zookeeper_ops_types.go [hunk 2]
```diff
@@ -48,12 +49,22 @@ type ZooKeeperOpsRequest struct {
 	Status            OpsRequestStatus        `json:"status,omitempty"`
 }
 
-// ZooKeeperOpsRequestSpec is the spec for ZooKeeperOpsRequest
+// +kubebuilder:validation:Enum=UpdateVersion;HorizontalScaling;VerticalScaling;VolumeExpansion;Restart;Reconfigure
+// ENUM(UpdateVersion, HorizontalScaling, VerticalScaling, VolumeExpansion, Restart, Reconfigure)
```
Comment (raihankhan) [inline apis/ops/v1alpha1/zookeeper_ops_types.go:53]: Remove Horizontal scaling and Reconfigure froim enum if not added in this PR.

**Review by raihankhan — CHANGES_REQUESTED** [review]

**Comment by AbdullahAlShaad** [comment]
Sign CLA

**Pull request merged**